exporter/azuremonitorexporter/testdata/config.yaml

```diff
@@ -0,0 +1,20 @@
+receivers:
+  examplereceiver:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  azuremonitor:
+  azuremonitor/2:
```

Comment:
Suggestion: we usually recommend making verbose comments here for each configuration option so users can just modify the example here to get started.

Reply:
good idea.